app/models/post.rb

```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
   belongs_to :user
+
+  def recent_comments(limit = 5)
+    comments.order(created_at: :desc).limit(limit)
+  end
+
+  def increment_comment_counter
+    update(comment_counter: comment_counter + 1)
+  end
+
+  def increment_like_counter
+    update(like_counter: like_counter + 1)
+  end
 end
```

Comment:
- [ ] Something similar happens here, the `Post` has many `Comments` and `Likes`. Please fix these relationships! 🤓 🔧 

- [ ] Also, according to the requirements, the methods that update both the `comments_counter` and `likes_counter` should be inside their respective models. 🤓 🔧 

![image](https://github.com/emarkees/Gist_Blog/assets/77069011/f5b6c58b-87e3-459a-8932-f71407f1e8c7)